protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -92,9 +94,23 @@ def estimate_block_height(current_block_height, current_block_timestamp, average
     AVG_BLOCK_TIME_FOR_ESTIMATE,
     UPDATE_2_TIME_UTC,
 )
+# Get current time in UTC
+current_utc_time = datetime.now(pytz.utc)
+# Add 4 days to the current time - use this as rough estimate for proposal pass time.
+estimated_proposal_pass_time = current_utc_time + timedelta(days=4)
+formatted_estimated_proposal_pass_time = estimated_proposal_pass_time.strftime("%Y-%m-%dT%H:%M:%S+00:00")
 
-print(f"Estimated block height for rewards multiplier update 1 {UPDATE_1_TIME_UTC} = {rewards_update_1_block}")
-print(f"Estimated block height for rewards multiplier update 2 {UPDATE_2_TIME_UTC} = {rewards_update_2_block}")
+proposal_pass_block_estimate = estimate_block_height(
```

Comment:
nit: would it be sufficient to just do the following to calculate the # of blocks to delay?

```
blocks to delay for update 1 = (time to update - time when proposal goes into gov) / avg block time
```

by this way, you prob don't need all the ref block height/time stuff

Reply:
Great idea!